protos/mission/mission.proto

```diff
@@ -182,6 +183,16 @@ message MissionItem {
         CAMERA_ACTION_START_VIDEO = 4; // Start capturing video
         CAMERA_ACTION_STOP_VIDEO = 5; // Stop capturing video
     }
+
+    enum PlaneAction {
+        VEHICLE_ACTION_NONE = 0;
+        VEHICLE_ACTION_ARM = 1;
+        VEHICLE_ACTION_DISARM = 2;
```

Comment:
I don't think we should have these in here. To me these are safety critical and should be explicitly used from the action plugin.